src/hosted-form/iframe-content/card-number-formatter.ts

```diff
@@ -1,5 +1,4 @@
-import { number } from 'card-validator';
-import creditCardType from 'credit-card-type';
```

Comment:
Don't need to pull in a separate package because 'card-validator' already has `creditCardType` (it is the same package).